### PR TITLE
Update to BYOK

### PR DESCRIPTION
**Pull request by toddham**
- Updated all bootstrap apps to latest AIProxySwift with BYOK
- Updated AppConstants files to include BYOK example
- Bug fix in Gemini call
- Update Button labels in image gen examples

### CODE REVIEW

File: AIProxyDeepL/AIProxyDeepL/AppConstants.swift [hunk 1]
```diff
@@ -9,12 +9,19 @@ import AIProxy
...
-    serviceURL: "service-url-from-your-developer-dashboard"
-)
+/* Uncomment for BYOK use cases */
+// let deepLService = AIProxy.deepLDirectService(
+//     unprotectedAPIKey: "your-deepL-key",
+//     accountType: .free
+// )
+
+/* Uncomment for all other production use cases */
+// let service = AIProxy.deepLService(
```
Comment (lzell) [inline AIProxyDeepL/AIProxyDeepL/AppConstants.swift:24]: service -> deepLService to match the block above

File: Demos/AIColorPalette/AIColorPalette/AIProxyIntegration.swift [hunk 2]
```diff
@@ -9,17 +9,24 @@ import AIProxy // The AIProxy SPM package is found at https://github.com/lzell/A
...
-    serviceURL: "service-url-from-your-developer-dashboard"
+#warning(
+    """
+    Uncomment one of the methods below. To build and run on device you must follow the AIProxy integration guide.
+    Please see https://www.aiproxy.pro/docs/integration-guide.html")
+    """
 )
 
+/* Uncomment for BYOK use cases */
+ let openAIService = AIProxy.openAIDirectService(
```
Comment (lzell) [inline Demos/AIColorPalette/AIColorPalette/AIProxyIntegration.swift:20]: Do you want to leave this commented like the other examples?

File: Demos/Chat/Chat/AppConstants.swift [hunk 3]
```diff
@@ -10,13 +10,19 @@ import AIProxy
...
     )
     
-    static let openAIService = AIProxy.openAIService(
-        partialKey: "hardcode_partial_key_here",
-        serviceURL: "hardcode_service_url_here"
-    )
+    /* Uncomment for BYOK use cases */
+//    static let openAIService = AIProxy.openAIDirectService(
+//        unprotectedAPIKey: "your-openai-key"
+//    )
```
Comment (lzell) [inline Demos/Chat/Chat/AppConstants.swift:21]: nit: Comment indent is at a different level than the block below

File: Demos/Classifier/Classifier/AppConstants.swift [hunk 4]
```diff
@@ -15,12 +15,19 @@ enum AppConstants {
...
     )
-    static let openAIService = AIProxy.openAIService(
-        partialKey: "hardcode_partial_key_here",
-        serviceURL: "hardcode_service_url_here"
-    )
+    
+    /* Uncomment for BYOK use cases */
+//     static let openAIService = AIProxy.openAIDirectService(
+//         unprotectedAPIKey: "your-openai-key"
+//     )
```
Comment (lzell) [inline Demos/Classifier/Classifier/AppConstants.swift:26]: Comment indent

File: Demos/EmojiPuzzleMaker/EmojiPuzzleMaker/ContentView.swift [hunk 5]
```diff
@@ -10,17 +10,24 @@ import SwiftUI
...
+    """
+    Uncomment one of the methods below. To build and run on device you must follow the AIProxy integration guide.
+    Please see https://www.aiproxy.pro/docs/integration-guide.html")
+    """
 )
 
+/* Uncomment for BYOK use cases */
+ let anthropicService = AIProxy.anthropicDirectService(
+     unprotectedAPIKey: "your-anthropic-key"
+ )
```
Comment (lzell) [inline Demos/EmojiPuzzleMaker/EmojiPuzzleMaker/ContentView.swift:23]: Could leave this commented for consistency with other examples

File: Demos/FilmFinder/FilmFinder/AppConstants.swift [hunk 6]
```diff
@@ -9,17 +9,23 @@ import AIProxy
...
     """
 )
 
-let groqService = AIProxy.groqService(
-    partialKey: "partial-key-from-your-developer-dashboard",
-    serviceURL: "service-url-from-your-developer-dashboard"
+/* Uncomment for BYOK use cases */
+let groqService = AIProxy.groqDirectService(
+    unprotectedAPIKey: "your-groq-key"
 )
```
Comment (lzell) [inline Demos/FilmFinder/FilmFinder/AppConstants.swift:23]: Same here

File: Demos/PuLIDDemo/PuLIDDemo/AppConstants.swift [hunk 7]
```diff
@@ -9,12 +9,18 @@ import AIProxy
...
     """
 )
 
-let replicateService = AIProxy.replicateService(
-    partialKey: "partial-key-from-your-developer-dashboard",
-    serviceURL: "service-url-from-your-developer-dashboard"
+/* Uncomment for BYOK use cases */
+let replicateService = AIProxy.replicateDirectService(
+    unprotectedAPIKey: "your-replicate-key"
 )
```
Comment (lzell) [inline Demos/PuLIDDemo/PuLIDDemo/AppConstants.swift:20]: Same here

File: Demos/Stickers/Stickers/AppConstants.swift [hunk 8]
```diff
@@ -10,12 +10,20 @@ import AIProxy
...
         """
     )
-    static let openAIService = AIProxy.openAIService(
-        partialKey: "hardcode_partial_key_here",
-        serviceURL: "hardcode_service_url_here"
+    
+    /* Uncomment for BYOK use cases */
+    static let openAIService = AIProxy.openAIDirectService(
+        unprotectedAPIKey: "your-openai-key"
     )
```
Comment (lzell) [inline Demos/Stickers/Stickers/AppConstants.swift:21]: Same here

File: Demos/Transcriber/Transcriber/AppConstants.swift [hunk 9]
```diff
@@ -23,13 +23,19 @@ enum AppConstants {
...
         """
     )
 
-    static let openAIService = AIProxy.openAIService(
-        partialKey: "hardcode_partial_key_here",
-        serviceURL: "hardcode_service_url_here"
+    /* Uncomment for BYOK use cases */
+    static let openAIService = AIProxy.openAIDirectService(
+        unprotectedAPIKey: "your-openai-key"
     )
```
Comment (lzell) [inline Demos/Transcriber/Transcriber/AppConstants.swift:34]: Same here

File: Demos/Translator/Translator/AppConstants.swift [hunk 10]
```diff
@@ -10,13 +10,19 @@ import AIProxy
...
         Please see https://www.aiproxy.pro/docs/integration-guide.html")
         """
     )
     
-    static let openAIService = AIProxy.openAIService(
-        partialKey: "hardcode_partial_key_here",
-        serviceURL: "hardcode_service_url_here"
+    /* Uncomment for BYOK use cases */
+    static let openAIService = AIProxy.openAIDirectService(
+        unprotectedAPIKey: "your-openai-key"
```
Comment (lzell) [inline Demos/Translator/Translator/AppConstants.swift:20]: Same here

File: Demos/Trivia/Trivia/AppConstants.swift [hunk 11]
```diff
@@ -10,12 +10,19 @@ import AIProxy
...
         Please see https://www.aiproxy.pro/docs/integration-guide.html")
         """
     )
-    static let openAIService = AIProxy.openAIService(
-        partialKey: "hardcode_partial_key_here",
-        serviceURL: "hardcode_service_url_here"
+
+    /* Uncomment for BYOK use cases */
+    static let openAIService = AIProxy.openAIDirectService(
+        unprotectedAPIKey: "your-openai-key"
```
Comment (lzell) [inline Demos/Trivia/Trivia/AppConstants.swift:20]: Same here

File: Demos/AIColorPalette/AIColorPalette/AIProxyIntegration.swift (hunk 2 above)
Comment (toddham) [reply Demos/AIColorPalette/AIColorPalette/AIProxyIntegration.swift:20]: Reason I left it uncommented is I get a bunch of errors for ex. "Cannot find 'openAIService' in scope"

Comment (lzell) [reply Demos/AIColorPalette/AIColorPalette/AIProxyIntegration.swift]: New approach of using the `#error` pragma is def better. 🚢

**Review by lzell — APPROVED** [review]

**Pull request merged**